component/client/chat/Message.tsx

```diff
@@ -18,25 +21,46 @@ class Message extends React.Component<IMessageProps, any> {
       backgroundColor: `${backgroundColor}`, // FIXME: I should not need to do this to the linter.
       color: fontColor
     };
+    const contextIdentifier = this.props.index + this.props.username;
```

Comment:
Why do we need an index?
Since there is only going to be one contextMenu at a time, you can just use a constant ID, right?